webpack.config.js

```diff
@@ -241,10 +241,6 @@ if ( calypsoEnv === 'desktop' ) {
 	// no chunks or dll here, just one big file for the desktop app
 	webpackConfig.output.filename = '[name].js';
 	webpackConfig.output.chunkFilename = '[name].js';
-} else {
-	// jquery is only needed in the build for the desktop app
-	// see electron bug: https://github.com/atom/electron/issues/254
-	webpackConfig.externals.push( 'jquery' );
```

Comment:
Does `@wordpress/components` still depend on jQuery?

Reply:
yep. `@wordpress/components` [needs `@wordpress/api-fetch`](https://github.com/WordPress/gutenberg/blob/master/packages/components/package.json#L28), and this one [needs jQuery](https://github.com/WordPress/gutenberg/blob/master/packages/api-fetch/package.json#L24)

Reply:
Complete aside, but if you type <kbd>y</kbd> on a GitHub file view, it'll give you a permanent link to the file like so: https://github.com/WordPress/gutenberg/blob/348e463b7cbdd246cf0744f6fa1459fefd0f37a7/packages/components/package.json#L28 (so the highlighted line remains the same if folks click on it in the future)

Reply:
Ah I totally misread the summary of https://github.com/WordPress/gutenberg/pull/7566, they're starting to move away from `jQuery` but it's still a dep